.github/CODEOWNERS

```diff
@@ -2,15 +2,31 @@
 # Each line is a file pattern followed by one or more owners.
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
+# default owners, overridden by package specific owners below
+* @asaj @cmcewen @timmoreton
+
 # directory and file-level owners. Feel free to add to this!
 /packages/analytics/ @nambrot @timmoreton
+/packages/attestation-service/ @nambrot
+/packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
-/packages/cli/ @mcortesi @asaj
+/packages/celotool/ @asaj
+/packages/cli/ @mcortesi
+/packages/contractkit/ @mcortesi
+/packages/dappkit/ @nambrot
+/packages/devutils/ @mcortesi
+/packages/docs/ @timmoreton
+/packages/faucet/ @mcortesi
 /packages/helm-charts/ @nambrot @timmoreton
+/packages/leaderboard/ @mrsmkl
+/packages/metadata-crawler/ @aaitor
 /packages/mobile/ @cmcewen @jmrossy
```

Comment:
should @i1skn  or @jeanregisser be here?

Reply:
You tell me :)